docs_src/getting-started/Ch-GettingStartedSDK-Go.md

```diff
@@ -89,28 +90,30 @@ With the device service application structure in place, time now to program the
     -   Change:
 
             example/cmd/device-simple/device-simple:
-              $(GO) build $(GOFLAGS) -o $@ ./example/cmd/device-simple
+              go mod tidy
+              $(GOCGO) build $(GOFLAGS) -o $@ ./example/cmd/device-simple
 
         to:
 
             cmd/device-simple/device-simple:
-              $(GO) build $(GOFLAGS) -o $@ ./cmd/device-simple
+              go mod tidy
+              $(GOCGO) build $(GOFLAGS) -o $@ ./cmd/device-simple
 
 4.  Save the file.
 
 5.  Enter the following command to create the initial module definition
     and write it to the go.mod file:
 
-        GO111MODULE=on go mod init
+        GO111MODULE=on go mod init github.com/edgexfoundry/device-simple
 
     ![image](EdgeX_GettingStartedGoModInit.png)
 
-6. Use an editor to open and edit the go.mod file created in ~/go/src/github.com/edgexfoundry/device-simple.  Add the code highlighted below to the bottom of the file.  This code indicates which version of the device service SDK and the associated EdgeX contracts module to use.
+6. Use an editor to open and edit the go.mod file created in ~/edgexfoundry/device-simple.  Add the code highlighted below to the bottom of the file.  This code indicates which version of the device service SDK and the associated EdgeX contracts module to use.
 
     ``` go
     require (
-        github.com/edgexfoundry/device-sdk-go v1.2.2
-        github.com/edgexfoundry/go-mod-core-contracts v0.1.58
+        github.com/edgexfoundry/device-sdk-go/v2 v2.0.0
+        github.com/edgexfoundry/go-mod-core-contracts/v2 v2.0.0
     )
     ```
     ![image](EdgeX_GettingStartedSDKGoMod.png)
```

Comment:
under !!! Note
move the URL from **[go.mod](https://github.com/edgexfoundry/device-sdk-go/blob/master/go.mod)** to **[the latest released version SDK](https://github.com/edgexfoundry/device-sdk-go)** , because we will rename master to main